### ISSUE TITLE
"Drivers" vs "Helpers" is not so helpful as a top-level distinction

### ISSUE THREAD

**Issue by dhalbert** [adafruit/Adafruit_CircuitPython_Bundle#419]
I find that dividing libraries between device "drivers" and "helpers" is not so helpful. I often have to think about which top-level category to look in. I would rather just get rid of this distinction.  For example, "Board-specific Helpers" could just be "Board Libraries".

Even the filename `drivers.rst` could be changed to `libraries.rst`.

**Comment by dastels** [comment]
Agreed. I'm constantly looking up bundle source and it's not always intuitive. There's a grey area that isn't a sensor but isn't is still hardware related, e.g. the portal code.
